README.md

```diff
@@ -138,7 +145,55 @@ You can use the provided setting with "info" log level.
 sudo cp scripts/log4j.properties /etc/gcs/conf
 ```
 
-## Create Policy
+## Deploy Ranger GCS Connector Adapter on Every Clusters
+
+The Ranger GCS connector adapter need to be executed by every worker nodes.
```

Comment:
"The Ranger GCS connector adapter needs to be installed on every node in the cluster." is enough, IMO.